src/renderer/components/dialogs/ReactionsDialog/index.tsx

```diff
@@ -81,19 +84,26 @@ function ReactionsDialogList({ reactionsByContact }: Props) {
     resolveContacts()
   }, [accountId, reactionsByContact])
 
+  const contactClickGen = (contactId: number) => {
+    return () => {
+      openViewProfileDialog(accountId, contactId)
+    }
+  }
+
   return (
     <ul className={styles.reactionsDialogList}>
       {contacts.map(contact => {
         return (
-          <li key={contact.id} className={styles.reactionsDialogListItem}>
+          <li
+            key={contact.id}
+            className={styles.reactionsDialogListItem}
+            onClick={contactClickGen(contact.id)}
+          >
```

Comment:
We should use `<button>` so it's accessible.
But I guess this can be done later, as part of Tauri.

Reply:
> We should use `<button>` so it's accessible.

I'm a bit rusty on that, will 'role="button"' be enough? I'd love to not increase hierarchy depth

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

> If using role="button" instead of the semantic <button> or <input type="button"> elements, you will need to make the element focusable and define event handlers for [click](https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event) and [keydown](https://developer.mozilla.org/en-US/docs/Web/API/Element/keydown_event) events. This includes handling the Enter and Space keypresses in order to process all forms of user input.

Reply:
i would keep such semantic changes out of this PR as well